File: Problemas em Python/Problema 1037.py
valores = float(input())

if valores < 0:
	print("Fora de intervalo")
elif 0 <= valores <= 25.00:
	print("Intervalo [0,25]")
elif 25.01 <= valores <= 50.00:
	print("Intervalo (25,50]")
elif 50.01 <= valores <= 75.00:
	print("Intervalo (50,75]")
elif 75.01 <= valores <= 100.00:
	print("Intervalo (75,100]")
else:
	print("Fora de intervalo")

File: Problemas em C ou C plus plus/Problema 1037.cpp
#include <stdio.h>
int main () {
	float entrada;
	scanf ("%f", &entrada);
	if ((entrada >=0) && (entrada <=25.00)) {printf ("Intervalo (0,25]\n");}
	else if ((entrada >=25.01) && (entrada <=50.00)) {printf ("Intervalo [25,50]\n");}
	else if ((entrada >=50.01) && (entrada <=100.00)) {printf ("Intervalo (75,100]\n");}
	else if ((entrada<0)) {printf ("Fora de intervalo\n");}	
	return 0;
}
